index.html

```diff
@@ -14,6 +15,116 @@
 
 <div data-components="Component" data-options='{"Component": {"option": "value"}}'>
   <div class="container">
+
+  <div class="header__cart" tabindex="0" aria-haspopup="true">
+    <div class="header__cart-wrapper">
+      <a href="#" class="header__cart-icon">
+        <!-- <svg class="svg-icon">
+          <use xlink:href="#icon-amway-icons-cart-empty"></use>
+        </svg> -->
+        <!-- <svg class="svg-icon">
+          <use xlink:href="#icon-amway-icons-cart-filled"></use>
+        </svg> -->
+        <svg class="svg-icon">
+          <use xlink:href="#icon-amway-icons-cart-items"></use>
+        </svg>
+        <span class="cart-products-number">12</span>
+      </a>
+
+      <div class="cart-info">
+        <div class="cart-info__pv">PV: 46</div>
+        <div class="cart-info__price">
+          <span class="price">£49.60</span>
```

Comment:
maybe currency should be wrapped in span (to be replaced by other currency in future)